test/acceptance/mixin-elems.html

```diff
@@ -1,50 +0,0 @@
-<link rel="import" href="../../../polymer/polymer.html">
```

Comment:
Since the classes now just use the mixins directly, I don't need these special custom elements to test them.